ibis/backends/clickhouse/datatypes.py

```diff
@@ -19,6 +20,10 @@
 )
 
 
+def _bool_type():
+    return getattr(getattr(ibis.options, "clickhouse", None), "bool_type", "Boolean")
```

Comment:
Isn't that always available in the configuration?

Reply:
`ibis.options.clickhouse` doesn't get registered until the module-level `__getattr__` in `ibis/__init__.py` is called with `'clickhouse'`. It's possible to parse a clickhouse type without ever having accessed a backend instance.

Will it ever be the case that a user runs code in `clickhouse/datatypes.py` without having accessed `ibis.clickhouse`? Unlikely. However, I do not want to have to type `ibis.clickhouse` to test this code locally.